docs/_docs/installation.md

```diff
@@ -1,138 +1,164 @@
 permalink: /docs/installation/
 ---
 
-Getting Jekyll installed and ready-to-go should only take a few minutes.
-If it ever becomes a pain, please [file an issue]({{ site.repository }}/issues/new)
-(or submit a pull request) describing the issue you
-encountered and how we might make the process easier.
+- [Requirements](#requirements)
+- [Install on macOS](#macOS)
+- [Install on Windows](../windows/)
+- [Upgrade Jekyll](#upgrade-jekyll)
 
-### Requirements
+Installing Jekyll should be straight-forward if your system meets the requirements.
+
+## Requirements
 
-Installing Jekyll should be straight-forward if all requirements are met.
 Before you start, make sure your system has the following:
 
-- GNU/Linux, Unix, or macOS
-- [Ruby](https://www.ruby-lang.org/en/downloads/) version 2.2.5 or above, including all development
-  headers (ruby installation can be checked by running `ruby -v`, development headers can be checked on Ubuntu by running `apt list --installed  ruby-dev`)
+- [Ruby](https://www.ruby-lang.org/en/downloads/) version 2.2.5 or above, including all development headers (ruby installation can be checked by running `ruby -v`)
 - [RubyGems](https://rubygems.org/pages/download) (which you can check by running `gem -v`)
 - [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/) (in case your system doesn't have them installed, which you can check by running `gcc -v`,`g++ -v`  and `make -v` in your system's command line interface)
 
-<div class="note info">
-  <h5>Problems installing Jekyll?</h5>
-  <p>
-    Check out the <a href="../troubleshooting/">troubleshooting</a> page or
-    <a href="{{ site.repository }}/issues/new">report an issue</a> so the
-    Jekyll community can improve the experience for everyone.
-  </p>
-</div>
+## Install on macOS {#macOS}
 
-<div class="note info">
-  <h5>Running Jekyll on Windows</h5>
-  <p>
-    While Windows is not officially supported, it is possible to get Jekyll running
-    on Windows. Special instructions can be found on our
-    <a href="../windows/#installation">Windows-specific docs page</a>.
-  </p>
-</div>
+We only cover latest macOS High Sierra 10.13 here, which comes with Ruby 2.3.3, older systems will need to [install a more recent Ruby version via Homebrew](#homebrew).
 
-## Install with RubyGems
+First, you need to [install Xcode from the App Store](https://itunes.apple.com/app/xcode/id497799835?mt=12) or [from Apple Developer website](https://developer.apple.com/xcode/). Then to install the command-line tools, open a terminal and run:
+
+```sh
+xcode-select --install
+```
 
-The best way to install Jekyll is via
-[RubyGems](https://rubygems.org/pages/download). At the terminal prompt,
-simply run the following command to install Jekyll:
+Check your Ruby version, and upgrade [RubyGems](https://rubygems.org/pages/download):
 
 ```sh
-gem install jekyll
+ruby -v
+2.3.3
+
+gem update --system
```

Comment:
Do we require a newer version of Rubygems outside of what's installed by default? If not, we shouldn't recommend that people upgrade Rubygems on their system.